tools/json_schemas_to_typescript.rb

```diff
@@ -61,7 +61,7 @@ def write_files(changed: nil, added: nil, removed: nil)
           if write_all || schema_path.in?(changed) || schema_path.in?(added)
             types_path = types_path(schema_path)
 
-            system(<<~SH.squish)
+            system(<<~SH.squish, exception: true)
```

Comment:
This would have made the issue resulting from https://github.com/glideapps/quicktype/issues/ 2753 more clear / immediately visible.